### ISSUE TITLE
Support for Windows Store

### ISSUE THREAD

**Issue by Kralizek** [countincognito/Zametek.ProjectPlan#7]
How about using the Windows Store bridge for a quick delivery of new versions?

**Comment by countincognito** [comment]
Certainly a good idea. I'd like PP to be more battle-hardened and feature rich before making it more readily available though.

**Comment by Kralizek** [comment]
oh, I already love it :)

**Comment by countincognito** [comment]
Now available:
https://apps.microsoft.com/detail/9MW5MDP78528

**Comment by Kralizek** [comment]
Lol, when I saw this PR being closed in the notifications, I didn't realize I was its initiator :D

**Comment by countincognito** [comment]
And it only took seven and a half years :-)

**Comment by Kralizek** [comment]
Does the store app supports free trials?

**Comment by countincognito** [comment]
No. But the free downloads will continue to be available. Hosting in the MS App Store isn't free, so I need to charge something to make it worthwhile.

**Comment by Kralizek** [comment]
I think a free trial would be beneficial. After all the store gives you much more visibility and a person that doesn't know of the repo would probably not put 60€ on something so niched.

**Comment by countincognito** [comment]
I'll see what I can do - it should be possible. But new releases can take between a few hours to a couple of days to get certified. Try checking back later.

**Comment by countincognito** [comment]
@Kralizek Okay - the free trial should now be active.
